### PR TITLE
first commit on branch gartee

### PR DESCRIPTION
**Pull request by gartee-john-PFG**
did a really cool feature...pls check it out

### CODE REVIEW

File: commitExercise.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+gartee line 1 commit 1
```
Comment (sousekgretchen-pfg) [inline commitExercise.txt:1]: looks good

Comment (kellyaultPFG) [inline commitExercise.txt:1]: best first line ever

Comment (barricktravisPFG) [inline commitExercise.txt:1]: looks GREAT

File: commitExercise.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+gartee line 2 commit 1
```
Comment (stickelabigailPFG) [inline commitExercise.txt:2]: Change to xyz.

Comment (sousekgretchen-pfg) [inline commitExercise.txt:2]: please change to first commit

File: commitExercise.txt (hunk 1 above)
Comment (stickelabigailPFG) [inline commitExercise.txt:1]: looks good

File: commitExercise.txt (hunk 2 above)
Comment (kellyaultPFG) [inline commitExercise.txt:2]: impressive, but may want to change first commit

**Review by stickelabigailPFG — CHANGES_REQUESTED** [review]
Reason for change

**Review by sousekgretchen-pfg — CHANGES_REQUESTED** [review]
reason for change

**Review by kellyaultPFG — CHANGES_REQUESTED** [review]
Change requested.

**Review by barricktravisPFG — APPROVED** [review]

**Review by barricktravisPFG — APPROVED** [review]

**Review by kellyaultPFG — APPROVED** [review]
approving

**Review by stickelabigailPFG — APPROVED** [review]
Great work!

**Pull request merged**